src/utils/selector.ts

```diff
@@ -189,4 +230,29 @@ export class Selector {
             .filter(([, value]) => /^#{3,}/.test(value))
             .map(([index, ]) => +index);
     }
+
+    private static handlePromptMetadata(metadatas: Map<RequestMetadata, string | undefined> , text: string) {
+        const promptVarDef = this.getPrompVariableDefinition(text);
+        if (promptVarDef) {
+            const varDefs = this.parsePromptMetadataForVariableDefinitions(metadatas.get(RequestMetadata.Prompt));
+            varDefs.push(promptVarDef);
+            metadatas.set(RequestMetadata.Prompt, JSON.stringify(varDefs));
+        }
+    }
+
+    private static async promptForInput(defs: PromptVariableDefinition[]): Promise<Map<string, string> | null> {
+        const promptVariables = new Map<string, string>();
+        for (const { name, description } of defs) {
+            const value = await window.showInputBox({
+                prompt: description ? `${description}` : `Input value for "${name}"`
```

Comment:
How about always make the prompt text as `Input value for "{name}"`, while setting the `description` as the `placeHolder` when description is available?

Reply:
That's a good idea! :)